ResourcesCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
```

Comment:
Can we derive `secondaryResources` and `primaryResources` from the `resources` state?

```suggestion
  const [resources, setResources] = useState([]);
  const primaryResources = resources[0];
  const secondaryResources = resources.
```